recurseFibonaci.c

```diff
@@ -0,0 +1,30 @@
+/*Rumeet Goradia - Recursive Fibonacci*/
```

Comment:
Nice job! You had one warning because of your fib function. The reason for this was because your conditionals don't cover all the possible input. To fix this, you just need to change the 2nd if statement to else if and the third to "else"